FinniversKit/Sources/DNA/Font/UIFont+FinniversKit.swift

```diff
@@ -3,118 +3,89 @@
 //
 
 import UIKit
+import Warp
 
-@objc public extension UIFont {
+@objc extension UIFont {
+    /// Font Medium/Bold with a size of 34 scaled for UIFontTextStyle.largeTitle
+    ///
     /// ## Usage:
     /// - It should only be used one T1 and it should be the first text element that catches the users attention.
     /// - It shall give the user an overview of which page he or she is located.
-    class var title1: UIFont {
-        Config.fontProvider.title1
+    /// - This always has the weight Medium.
+    public static var title1: UIFont {
+        Warp.Typography.title1.uiFont
     }
 
+    /// Font Medium/Bold with a size of 28 scaled for UIFontTextStyle.title1
+    ///
     /// ## Usage:
     /// - A page can contain multiple T2 to divide content into several sections.
     /// - There should be a lot of space between sections to create a clear distinction on the content.
-    class var title2: UIFont {
-        Config.fontProvider.title2
-    }
-
-    /// ## Usage:
-    /// - A page can contain multiple T2 to divide content into several sections.
-    /// - There should be a lot of space between sections to create a clear distinction on the content.
-    class var title2Strong: UIFont {
-        Config.fontProvider.title2Strong
+    /// - This always has the weight Light.
+    public static var title2: UIFont {
+        Warp.Typography.title2.uiFont
     }
 
+    /// Font Medium/Bold with a size of 22 scaled for UIFontTextStyle.title2
+    ///
     /// ## Usage:
     /// - If a T2 have more sublevels, you can use T3.
-    class var title3Strong: UIFont {
-        Config.fontProvider.title3Strong
+    /// - This always has the weight Light.
+    public static var title3: UIFont {
+        Warp.Typography.title3.uiFont
     }
 
+    /// Font Medium/Bold with a size of 16 scaled for UIFontTextStyle.callout
+    ///
     /// ## Usage:
-    /// - If a T2 have more sublevels, you can use T3.
-    class var title3: UIFont {
-        Config.fontProvider.title3
+    /// - This have the same size as the body text, but is always bolded (Medium) to differenciate them.
+    public static var bodyStrong: UIFont {
+        Warp.Typography.title4.uiFont
```

Comment:
just checking ...        ` Config.fontProvider.title3 ==  Warp.Typography.title4.uiFont`

Reply:
Probably it shows it like that because we removed `title3Strong` and we only have `title3`, and `bodyStrong` is `title4`. So the result is okey :D the change log is a bit strange.